lms/djangoapps/course_home_api/progress/v1/tests/test_views.py

```diff
@@ -0,0 +1,61 @@
+"""
+Tests for Progress Tab API in the Course Home API
+"""
+
+from datetime import datetime
+import ddt
+
+from django.urls import reverse
+
+from course_modes.models import CourseMode
+from lms.djangoapps.course_home_api.tests.utils import BaseCourseHomeTests
+from lms.djangoapps.course_home_api.toggles import COURSE_HOME_MICROFRONTEND
+from openedx.features.content_type_gating.models import ContentTypeGatingConfig
+from student.models import CourseEnrollment
+
+
+@ddt.ddt
+class ProgressTabTestViews(BaseCourseHomeTests):
+    """
+    Tests for the Progress Tab API
+    """
+
+    @classmethod
+    def setUpClass(cls):
+        BaseCourseHomeTests.setUpClass()
+        cls.url = reverse('course-home-progress-tab', args=[cls.course.id])
+        ContentTypeGatingConfig.objects.create(enabled=True, enabled_as_of=datetime(2017, 1, 1))
```

Comment:
I think this was all changed to be using the `setUp` method. See https://github.com/edx/edx-platform/blob/master/lms/djangoapps/course_home_api/dates/v1/tests/test_views.py#L22 for an example.